### PR TITLE
Change link to xref to solve problems

### PR DESCRIPTION
**Pull request by melcorr**
The xref seems to solve our issues. 

Cherry-pick into:

* [ ] Foreman 3.0
* For Foreman 2.5, file a separate PR request

### CODE REVIEW

**Comment by melcorr** [comment]
This all builds locally for me everywhere :(

**Comment by melcorr** [comment]
I think it's the hard-coding of the context.

**Comment by melcorr** [comment]
i'm just going to mess around as texting locally is not working

File: guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ For more information, see xref:Creating_a_Custom_File_Type_Repository_{context}[
 For more information, see {AdministeringDocURL}sect-Administering-Installing_the_Katello_Root_CA_Certificate[Installing the Katello Root CA Certificate] in the _Administering {ProjectName}_ guide.
 . An Organization Debug Certificate.
 ifndef::satellite[]
-For more information, see {ManagingOrganizationsLocationsDocURL}Creating_an_Organization_Debug_Certificate_managing-organizations-locations[].
+For more information, see {ManagingOrganizationsLocationsDocURL}Creating_an_Organization_Debug_Certificatee_{context}[].
```
Comment (maximiliankolb) [inline guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc:16]: We have an extra`e` & the `{context}` depends on the context of the link, whereas the context of the anchor defines the anchor. > We cannot "mix" `{AnyDocURL}` with `{context}`.

**Comment by maximiliankolb** [comment]
IMHO this cannot due to the design: we're checking links against docs.theforeman.org before deploying the master branch to docs.theforeman.org. I think the pipeline for master branch fails due to a link that cannot yet exist yet, because we're adding it with the commit of the pipeline. see https://github.com/theforeman/foreman-documentation/runs/4549908867?check_suite_focus=true#step:11:31

**Comment by maximiliankolb** [comment]
I obviously would support if you revert my commit. 

On the other hand, we need a way to add new guides which are linked to without breaking our pipeline.

**Comment by melcorr** [comment]
So if I just revert your link to, add another PR, it will be OK?

File: guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc (hunk 1 above)
Comment (melcorr) [reply guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc:16]: Ahhhhh
I get it

Comment (melcorr) [reply guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc:16]: Thank you. I'm tired and was hoping for a quick fix.

**Pull request closed without merging**